scipy/sparse/linalg/_expm_multiply.py

```diff
@@ -447,7 +408,7 @@ def d(self, p):
         where :math:`||.||` is the 1-norm.
         """
         if p not in self._d:
-            est = _onenormest_matrix_power(self._A, p, self._ell)
+            est = onenormest(self._A**p, rng=self._rng)
```

Comment:
I suppose `self._A` is already a linear operator, so the call to `aslinearoperator` that was in `_onenormest_matrix_power` is not needed.